dockerfiles/cuda118/Dockerfile

```diff
@@ -13,4 +13,4 @@ RUN pip3 install --no-cache-dir -r /tmp/requirements.txt --extra-index-url https
 COPY ./src /workdir/src
 WORKDIR /workdir
 
-ENTRYPOINT [ "python3", "-m", "src.train" ]
+ENTRYPOINT [ "python3", "-m" ]
```

Comment:
I think you should add a 

CMD [ "src.train" ]

here as well, so it defaults to it, it has been added to the 120 Dockerfile, but ommited in this one

Reply:
I've tried it but it won't work when you pass extra arguments.

If you set it and just run `docker run <image_name>`, all works as expected.

But if you run with extra args `docker run <image_name> --batch 32 --width 320`, docker will overwrite the CMD so the final command will only be `python3 -m --batch 32 --width 320`.

Reply:
yes, but what i wanted to highlight is that now there is a discrepancy between the cuda120 and cuda118 dockerfiles, i don't think it's intentional

Reply:
eh forgot ctrl+s